src/js/components/PageComponents.js

```diff
@@ -263,13 +263,18 @@ export class GeoDashNavigationBar extends React.Component {
                         >
                             <span className="navbar-toggler-icon"></span>
                         </button>
-                        <div className="collapse navbar-collapse" id="navbarSupportedContent">
-                            <ul id="login-info" className="navbar-nav mr-auto">
+                        <div className="collapse navbar-collapse justify-content-between" id="navbarSupportedContent">
+                            <ul className="navbar-nav">
                                 <li className="nav-item my-auto">
                                     <h1>GEO-DASH</h1>
                                 </li>
                             </ul>
-                            <ul className="navbar-nav mr-0">
+                            <ul className="navbar-nav" style={{flex: 1, justifyContent: "flex-end"}}>
+                                {uri !== "/widget-layout-editor" &&
```

Comment:
Your JSX below is the inverse logic.  Two options:
1. check for `plotId > 0` 
2. Use a Ternary so you have one conditional with two outcomes together.